pytest_mypy_plugins/tests/test-regex_assertions.yml

```diff
@@ -6,3 +6,11 @@
 
     reveal_type(a)  # N: Revealed type is "builtins.int"
     reveal_type(b)  # N: .*str.*
+
+- case: expected_message_regex_with_out
```

Comment:
We also need a test without `regex` attribute and with `regex: no` where we try to use regex outout

Reply:
I'm not sure how to do negative tests - ie tests where we expect a failure fail. Are there any examples of this kind of test?

Reply:
For example, if I added this test:

```yaml
- case: should_fail
  regex: no
  main: |
    a = 'hello'
    reveal_type(a)  # N: .*str.*
```

I'd expect the test to fail - but that fails the build. Is there a way to express that I _expect_ this to fail?

Reply:
Nope, see #46 

The easiest way for now is to add all failing tests into a single non-collectable file. And run it with `pytest` after the main test case. We can `grep` that `failed X tests`

Reply:
Can you approve me to run workflows? I have something which works locally, but I'd like to see if it works in CI.